webapp/views/App/views/Designer/TaxonomyList/Row/useTaxonomyRow.js

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { matchPath, useLocation, useHistory } from 'react-router'
+
+import * as ObjectUtils from '@core/objectUtils'
+import * as Survey from '@core/survey/survey'
+import * as Taxonomy from '@core/survey/taxonomy'
+
+import { useI18n } from '@webapp/store/system'
+import { useSurveyInfo } from '@webapp/store/survey'
+import { DialogConfirmActions, NotificationActions } from '@webapp/store/ui'
+
+import { appModuleUri, designerModules } from '@webapp/app/appModules'
+
+import * as TaxonomyActions from '@webapp/loggedin/surveyViews/taxonomy/actions'
+
+const useTaxonomyRow = (props) => {
+  const { row: taxonomy, canSelect, onSelectTaxonomy, selectedItemUuid } = props
+
+  const [deleted, setDeleted] = useState(false)
```

Comment:
is it necessary to keep track of the deletion state? What about the pagination in the list? don't we need to load again the taxonomies in the shown page once a taxonomy has been deleted?

Reply:
this is the reason which I add this deleted state in order to avoid the "re-fetch" but if you propose other behavior I will do

Reply:
I think refetching with current offset is necessary after deleting the taxonomy.